README.md

```diff
@@ -127,9 +127,9 @@ $ rails generate shopify_app:install --api_key <your_api_key> --secret <your_app
 ```
 
 Other options include:
-* `application_name` - the name of your app, it can be supplied with or without double-quotes if a whitespace is present. (e.g. `--application_name Example App` or `--application_name "Example App"`)  
-* `scope` - the Oauth access scope required for your app, eg **read_products, write_orders**. *Multiple options* need to be delimited by a comma-space, and can be supplied with or without double-quotes  
-(e.g. `--scope read_products, write_orders, write_products` or `--scope "read_products, write_orders, write_products"`)  
```

Comment:
The whitespaces at the end of ln#131 and #132 were deliberate. They were included to *force* the `(e.g. ` and `For more..` onto the following line when the markdown document is rendered. 
I'm okay with the deletion on ln#130.